Assets/Scripts/UIScripts/PauseScreen.cs

```diff
@@ -43,13 +46,20 @@ public void Resume()
     {
         pauseMenuUI.SetActive(false);
         settingsMenuUI.SetActive(false);
-        //MidiManager.current.ResumePlayback();
+        countdownManager.SetCountdown();
+        StartCoroutine(ResumeAfterCountdown());
+    }
+
+    public IEnumerator ResumeAfterCountdown()
+    {
+        yield return new WaitForSecondsRealtime(5);
+            
+        Time.timeScale = 1f;
         if (Time.time > 5)
```

Comment:
Hmm why is this Time.time and not Time.timeSinceLevelLoad? 🤔
Also do we need this if-statement? Who made this chang-oh I did lol: https://github.com/jamilfaisal/Meow-Tunes/commit/bb98ca9e2010eda1d8fe8c0446317e788085ea2c

"Fix bug of music playing in the first 5 seconds when player resumes". I think this if-statement isn't relevant anymore since we now wait 5 seconds before this line